Samples/kotlin/src/main/java/com/phunware/kotlin/sample/VoicePromptActivity.kt

```diff
@@ -205,7 +202,26 @@ class VoicePromptActivity : AppCompatActivity(), OnPhunwareMapReadyCallback,
         }
 
         if (voiceEnabled) {
-            textToVoice(displayHelper.stringForDirection(this, maneuver))
+            val pair = navOverlay.getManeuverPair()
+            var text = displayHelper.stringForDirection(this, pair.mainManeuver)
+
+            val turnManeuver  = pair.turnManeuver
+            var turnable : Boolean  = false
+            if(turnManeuver != null)
+            {
+                turnable = turnManeuver.isTurnManeuver
+            }
+
+            if(turnManeuver != null && position < navigator.maneuvers.size - 1 && turnable)
+            {
+                text += " then "
+                text += displayHelper.stringForDirection(this, turnManeuver)
+            }
+            else
+            {
+                text += " to arrive at the destination."
```

Comment:
Can we move these strings to strings.xml

Reply:
done